test/functional/DDR_Test/src/j9vm/test/ddrext/junit/TestThread.java

```diff
@@ -238,18 +222,19 @@ private String getStackSlotsValues(String stackslotOutput) {
 		return results;
 	}
 
-	/*
-	 * If needELS, we return the second param list for stackslots:
-	 * thread,sp,a0,pc,literals,elsElse,we return the first param list for
-	 * stackslots : thread,sp,a0,pc,literals
+	/**
+	 * Constructs the parameter list of stackslots (thread, sp, a0, pc, literals) from the output.
+	 * @param stackOutput the output of the stackslots command
+	 * @param threadAddress the address of the walked thread
+	 * @return the parameter list for stackslots, or null if it failed to construct the list
 	 */
-	private String getStackSlotsParamSet(String stackOutput,
-			String threadAddress, boolean needELS) {
+	private static String getStackSlotsParamSet(String stackOutput,
+			String threadAddress) {
 		String output[] = stackOutput.split(Constants.NL);
 		String paramList = threadAddress;
 		for (String aLine : output) {
 			if (aLine.contains("Initial values")) {
-				String sp = null, pc = null, literals = null, a0 = null, els = null;
+				String sp = null, pc = null, literals = null, a0 = null;
 				String tokens[] = aLine.split(",");
 				for (int i = 0; i < tokens.length; i++) {
 					String token = tokens[i].trim();
```

Comment:
Please correct the declaration of `tokens`, use an enhanced for loop and use `startsWith()` consistently:
```java
				String[] tokens = aLine.split(",");
				for (String token : tokens) {
					token = token.trim();
					if (token.startsWith("walkSP")) {
						sp = token.split("=")[1].trim();
					} else if (token.startsWith("PC")) {
						pc = token.split("=")[1].trim();
					} else if (token.startsWith("literals")) {
						literals = token.split("=")[1].trim();
					} else if (token.startsWith("A0")) {
						a0 = token.split("=")[1].trim();
					}
				}
```